lib/src/Uncertainty/Algorithm/MetaModel/Kriging/KrigingResult.cxx

```diff
@@ -541,7 +541,7 @@ CovarianceMatrix KrigingResult::getConditionalCovariance(const Point & point) co
   const Point data(point);
   // 1) compute \sigma_{x,x}
   LOGINFO("Compute interactions Sigma_xx");
-  const CovarianceMatrix sigmaXX(covarianceModel_(Point(inputDimension, 0.0)));
+  const SquareMatrix sigmaXX(covarianceModel_(Point(inputDimension, 0.0)));
```

Comment:
Wrong here should be `CovarianceMatrix`

Reply:
I merged this too quickly, how do we fix this ?

Reply:
Yep.

Reply:
@jschueller  I have added a commit on my branch josephmure:covmodel_bugfix that applies Sofiane's corrections (this one and the one below). If you want, you can pick it in your branch.

Reply:
if you already got this in a branch, could you submit it ?

Reply:
OK.